.github/workflows/main.yml

```diff
@@ -38,10 +32,22 @@ jobs:
           - python-version: "3.11"
             env:
               TOXENV: latest
-          - python-version: "3.11"
+          - python-version: "3.12"
+            env:
+              TOXENV: py
+          - python-version: "3.12"
+            env:
+              TOXENV: latest
+          - python-version: "3.13"
+            env:
+              TOXENV: py
+          - python-version: "3.13"
+            env:
+              TOXENV: latest
+          - python-version: "3.13"
             env:
               TOXENV: twinecheck
-          - python-version: "3.11"  # Keep in sync with tox.ini
+          - python-version: "3.13"  # Keep in sync with tox.ini
```

Comment:
Not sure why does this mention tox.ini as we usually need to sync this with `.readthedocs.yml` (which wasn't changed in this PR, but we currently can't use 3.13 to build docs as RTD doesn't support that yet, so the docs Python version should be bumped only to 3.12.